apps/internal/oauth/ops/authority/authority.go

```diff
@@ -213,9 +214,23 @@ func NewInfoFromAuthorityURI(authorityURI string, validateAuthority bool) (Info,
 		return Info{}, err
 	}
 
+	port := u.Port()
```

Comment:
Does this not always come up as 443 ? (a unit test should cover it)

Reply:
for private cloud STS comes up at port 3001